index.html

```diff
@@ -297,6 +310,43 @@ <h2>The VC HTTP API</h2>
       <h3>Authorization</h3>
       <p>
       </p>
+
+      <p class="issue" title="Apply group resolution">
+Convert the following group resolution into specification text:<br><br>
+<a href="https://w3c-ccg.github.io/meetings/2021-07-13-vchttpapi/#resolution-1">
+RESOLUTION</a>: The VC HTTP API work item group will separate GNAP from OAuth2
+until it is clear how much extra work GNAP would add within the scope of the
+specification.
+      </p>
+
+      <p class="issue" title="Apply group resolution">
+Convert the following group resolution into specification text:<br><br>
+<a href="https://w3c-ccg.github.io/meetings/2021-07-13-vchttpapi/#resolution-2">
+RESOLUTION</a>: One of the authorization mechanisms defined for the VC-HTTP-API
+MUST be OAuth 2 Bearer tokens.
+      </p>
+
+      <p class="issue" title="Apply group resolution">
+Convert the following group resolution into specification text:<br><br>
+<a href="https://w3c-ccg.github.io/meetings/2021-07-13-vchttpapi/#resolution-3">
+RESOLUTION</a>: How a VC HTTP API server validates an authorization token is out
+of scope.
+      </p>
+
+      <p class="issue" title="Apply group resolution">
+Convert the following group resolution into specification text:<br><br>
+<a href="https://w3c-ccg.github.io/meetings/2021-07-13-vchttpapi/#resolution-4">
+RESOLUTION</a>: One of the authorization protocols that will be defined for use
+in the VC-HTTP-API MUST be OAuth 2 Client Credentials.
+      </p>
+
+      <p class="issue" title="Apply group resolution">
```

Comment:
This resolution was passed with insufficient consensus. Contrary to all other CCG meetings, someone made a decision that "majority" voting would be sufficient for resolution approval. I objected at the time of the resolution and was told the decision had been made to use majority vote and so that's what we were doing for that meeting. 

Since the entirely of my previous experience with the W3C has always enabled either email or regular calls as a forum for individuals to register objections to decisions made in task forces and smaller subgroups, the lack of any mechanism for the several people on the call who voted against this resolution constitutes a process failure at best, one that, frankly, looks suspiciously like someone set up the majority proposal to intentionally push through their preferred features.

To avoid the appearance railroading this decision, I'd like it removed from this PR. If desired, a new issue could be created to establish whether or not the CCG as a whole supports this resolution. Or this can be brought up in a call where we rely on more standard W3C notions of consensus development.

TL;DR: when the group is clearly bifurcated for and against a resolution, 50+% is not an appropriate decision making tool. Please remove.

Reply:
FWIW, Community process guidelines are at https://w3c-ccg.github.io/workitem-process/ which sadly, mentions the "CG's documented consensus process" but doesn't provide a link.

It does make it clear that in the adoption stage and final report stage that "principled objections" are sufficient to halt the adoption of a work item.

Reply:
To clarify: is the resolution that you're objecting to this one: "RESOLUTION: One of the authorization mechanisms defined for the VC HTTP API ￼MUST be GNAP key-bound access tokens."

Reply:
Yes. If there were a particular mechanism proposed we could evaluate it on its merit. However, we don't know what mechanism would be required. Making GNAP a MUST seems inappropriate at this stage.

That doesn't mean GNAP won't eventually be supported, just that we don't, today, have consensus that we should bind the VC-HTTP-API to unknown features outside our direct control.